src/makeRobotAccountant.js

```diff
@@ -1,12 +1,27 @@
 'use strict';
 
-/**
- *
- * @return {function}
- */
-
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const f = (a) => {
+    count++;
+
+    let sum = a;
+
+    const c = (b) => {
+      sum += b;
+
+      if (count >= 4 && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return sum;
+    };
+
+    return c;
+  };
```

Comment:
try to use more meaningful names for variables - it is hard to understand such code